draft-wang-ppm-differential-privacy.md

```diff
@@ -451,9 +465,81 @@ class DpMechanism:
 
 > TODO: Specify a Gaussian sampler
 
-## Randomized Response {#rr}
+## Binary Randomized Response {#rr}
+
+The binary randomized response mechanism specified here aims to provide the
+DP guarantee at the Client level. It is the same as the one specified in
+Appendix C.1 of {{MJTB+22}}.
+
+To apply noise at Client side, the Client flips a coin at each coordinate of its
+vector. If the coin turns out to be a head, the Client will change the value `x`
+at that coordinate to `1 - x`. The probability of the coin giving a head is
+`1 / (e^(EPSILON_0) + 1)`, where `EPSILON_0` provides the pure `EPSILON_0`-DP
+guarantee, in the deletion-DP model (Definition II.4 of {{EFMR+20}}), which
+ensures that it is impossible to confidently determine whether an individual
+contributed data to the final aggregate result.
+
+The final aggregate result is obtained by "debiasing" the aggregation of the
+noisy Client measurements. It is expressed as a vector of floats, because the
+debiasing operations involve floating point arithmetics.
+
+Since the noise generated by each Client at each coordinate is independent, and
+based on a coin flip with probability `1 / (e^(EPSILON_0) + 1)`, the noise
+generated by `n` number of Clients at each coordinate is a binomial
+distribution. By Central Limit Theorem (CLT), as `n` grows, the noise
```

Comment:
CLT is probably folklore enough ("folklore" == "well-known enough to not require a citation") but we should probably state it for the reader in the conventions section of this draft, at least informally. (Similar to how we handle `e`.)